client/cmdhf14a.c

```diff
@@ -9,33 +9,10 @@
 // High frequency ISO14443A commands
 //-----------------------------------------------------------------------------
 
-#include <stdio.h>
-#include <stdlib.h>
-#include <inttypes.h>
-#include <string.h>
-#include <unistd.h>
-#include "util.h"
-#include "util_posix.h"
-#include "iso14443crc.h"
-#include "data.h"
-#include "proxmark3.h"
-#include "ui.h"
-#include "cmdparser.h"
 #include "cmdhf14a.h"
-#include "common.h"
-#include "cmdmain.h"
-#include "mifare.h"
-#include "cmdhfmfu.h"
-#include "mifarehost.h"
```

Comment:
Moving all includes to iso14443a.h has the effect that they are also included by every module which needs functions from iso14443a.c and therefore includes iso14443a.h. While including every possible *.h file eliminates the need to think about what is really required, it also has some negative effects:
* changing any of these leads to compiling virtually everything when using make.
* An unchanged *.c file may suddenly not compile anymore just because some unrelated *.h file has changed.

But this is just my opinion.  It is hard to discuss coding style.